articles/concepts/Glossary.md

```diff
@@ -12,31 +12,33 @@ uid: microsoft.quantum.glossary
-|Measurement|Obtaining a classical bit from a qubit (or set of qubits). See the [Qubit Concepts](xref:microsoft.quantum.concepts.qubit) section for more details.|
-|Mutable|A variable whose value may be changed after it is created.|
-|Namespace|A label for a collection of related names (typically operations, functions, and types). For instance the namespace [`Microsoft.Quantum.Preparation`](xref:microsoft.quantum.preparation) labels all of the symbols defined in the standard library that help with preparing initial states.|
-|Operation|The basic unit of quantum execution in Q#. It is roughly equivalent to a function in C or C++ or Python, or a static method in C# or Java.|
-|Operator Application|Performing a quantum operation. This typically applies a unitary matrix to the current state vector. See [Introduction to Quantum Concepts](xref:microsoft.quantum.concepts.intro) for more detail.|
-|Oracle|A subroutine that provides data-dependent information to a quantum algorithm at runtime. Typically, the goal is to provide a superposition of outputs corresponding to inputs that are in superposition.   |
-|Partial Application|Calling a function or operation without all the required parameters. The returns a new callable that only needs the missing parameters supplied during a future application.|
-|Pauli Operators|The `X`, `Y` and `Z` quantum gates.|
-|Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level.|
-|Quantum Circuit|The representation of a program for a quantum computer. See the <xref:microsoft.quantum.concepts.circuits> section for more details.|
-|Quantum State|A representation of the qubits in the system. This is usually denoted as a complex column vector. For more information, see <xref:microsoft.quantum.concepts.vectors>. |
-|Qubit|Unit of quantum storage. See the <xref:microsoft.quantum.concepts.qubit> section for more details.|
-|Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will re-try until successful (or a limit has been reached). |
-|Target machine|A compilation target that lowers an abstract quantum program towards hardware or simulation. This typically include re-writes for many purposes including gate replacement, encoding for error correction, geometric layout and others.|
-|Tuple|Comma separated types grouped together via parenthesis. |
-|User-defined type|Collection of built-in or previously defined types that may be referred to as a single unit.|
-
+|<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by dagger symbol, for example the adjoint of vector $v$ is $v^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|
```

Comment:
```suggestion
|<a id="adjoint"></a>Adjoint|The complex conjugate transpose of an operation. For operations that implement a unitary operator, the adjoint is the inverse of the operation. Adjoints are indicated by the dagger symbol, for example, the adjoint of an operator $A$ is $A^\dagger$. For more information, see [Adjoint](xref:microsoft.quantum.language.file-structure#adjoint).|
```
Although the adjoint of a vector is mathematically well defined, I think it is better to use operators in the example, since they are more relevant in the context of quantum computing.

Reply:
updated